e2e/Makefile

```diff
@@ -9,20 +9,22 @@ QUAY_NAMESPACE ?= workspaces
 OPERATOR_IMG ?= quay.io/$(QUAY_NAMESPACE)/workspaces-op:test-${DATE_SUFFIX}
 SERVER_IMG ?= quay.io/$(QUAY_NAMESPACE)/workspaces-rest:test-${DATE_SUFFIX}
 CONCURRENCY ?= 1
+IMAGE_BUILDER ?= docker
+export IMAGE_BUILDER
```

Comment:
is this needed? is it also needed in server/Makefile?

Reply:
I want to pass on `IMAGE_BUILDER` to all subinvocations of make, and I could either:
- `export IMAGE_BUILDER` as an environment variable
- pass it as a parameter every time we invoke make

I went with the first option, since it was easier for me to set up & test, but I can change it to the second option if you want.